test/google_scraping/dashboard/keyword_scarper_worker_test.exs

```diff
@@ -0,0 +1,30 @@
+defmodule GoogleScraping.Dashboard.KeywordScraperWorkerTest do
+  use GoogleScraping.DataCase, async: true
+
+  alias GoogleScraping.Dashboard.{Keywords, KeywordScraperWorker}
+
+  describe "perform/1" do
+    test "given keyword, download and store HTML" do
+      use_cassette "google/cat" do
+        keyword = insert(:keyword, name: "cat")
+
+        Oban.insert(KeywordScraperWorker.new(%{keyword_id: keyword.id}))
+        updated_keyword = Keywords.get_keyword_by_id!(keyword.id)
+
+        assert updated_keyword.status == :completed
+        assert updated_keyword.html != nil
+      end
+    end
+
+    test "given error for keyword search, fails job" do
+      stub(HTTPoison, :get, fn _, _ -> {:error, %{reason: :error}} end)
```

Comment:
We prefer using the `expect` method as we would know that this method is called, or otherwise would raise an error when it isn't 😄 
https://nimblehq.co/compass/development/code-conventions/elixir/ex-unit/#module-mocking

Reply:
So many conventions to keep in mind 😓 

Thanks! It was fixed here: https://github.com/Nihisil/nimble-certification-elixir/pull/41/commits/03c2b57cd07d71175fed403065e39d1846131ed4